Community/News/20190130_welcome_graal.md

```diff
@@ -0,0 +1,34 @@
+# Welcome Graal
+
+Currently, [GrimoireLab](https://github.com/chaoss/grimoirelab) allows to produce analytics with data extracted from more than 30 tools related with contributing to Open Source development such as version control systems, issue trackers and forums. Despite the large set of metrics available in GrimoireLab, none of them relies on information extracted from source code, thus limiting the end-users to benefit of a wider spectrum of software development data.
+
+[Graal](https://github.com/Bitergia/graal) is a tool that allows to conduct customizable and incremental analysis of source code by leveraging on existing tools. It enhances [Perceval](https://www.researchgate.net/publication/325334393_Perceval_Software_Project_Data_at_Your_Will) (one of the key GrimoireLab‘s components) and produces an output similar to Perceval one to ease the integration with GrimoireLab, thus complementing the analytics offered by latter with source Code related metrics.
```

Comment:
For Perceval, I would include a link to its repository instead of or in addition to this reference.